### ISSUE TITLE
Unable to start correctly

### ISSUE THREAD

**Issue by floatingshed** [rg3/youtube-dl#13505]
Please advise on the required dependencies for windows 10 64 bit. I have installed Microsoft Visual C++ 2010 Redistributable Package (x86) but cannot use any version of youtube-dl after 2016-05-21.2, later versions fail with "application was unable to start correctly (oxc000007b)". Thanks

**Comment by AndrewKiri** [comment]
Looks like it misses some dependencies. I was able to find this thread on [SO](https://stackoverflow.com/questions/10492037/the-application-was-unable-to-start-correctly-0xc000007b), which recommends to use dependency walker to see what is missing. Have you considered giving it a try?

**Comment by floatingshed** [comment]
Yes I have tried it but I'm afraid that I don't understand the result. There are hundreds of errors.
Clearly some dependencies changed between versions: 2016-05-21.2 and 2016-05-30.2 as the former works but the latter (and all later versions) fails. The changelog doesn't go back that far, I checked there for clues!
